packages/vscode-ui5-language-assistant/src/extension.ts

```diff
@@ -125,6 +150,101 @@ function updateCurrentModel(model: UI5Model | undefined) {
   }
 }
 
+async function showWebView() {
```

Comment:
try to refactor this 70 lines function by extracting logical sub-parts
so the high level logical flow would be easily visible